client/shipping-label/views/history.js

```diff
@@ -0,0 +1,79 @@
+import React, { PropTypes } from 'react';
+import CalypsoI18n from 'i18n-calypso';
+import Gridicon from 'components/gridicon';
+import formatDate from 'lib/utils/format-date';
+import { translate as __ } from 'lib/mixins/i18n';
+import { sprintf } from 'sprintf-js';
+const LabelHistory = ( { labelsHistory } ) => {
+	if ( ! labelsHistory || ! labelsHistory.length ) {
+		document.getElementById( 'woocommerce-order-label-history' ).style.display = 'none';
+		return null;
+	}
+	document.getElementById( 'woocommerce-order-label-history' ).style.display = 'block';
```

Comment:
I think we should be hiding the history metabox by default (in our stylesheet) and overriding the `display` property if there's history to be shown. I'm willing to concede the class-based approach, but are you *sure* that using a class to set `display` isn't cross browser compatible?

See: https://developer.mozilla.org/en-US/docs/Web/CSS/display#Browser_compatibility

I'd also like to see this logic moved to where the `LabelHistory` component is actually hooked into the DOM. (in the `ShippingLabelRootView` component)

(cc: @allendav @DanReyLop @v18 @nabsul for discussion)